commands/soup.js

```diff
@@ -0,0 +1,20 @@
+const talkedRecently = new Set();
+module.exports = {
+    name: 'soup',
+    description: "soup",
+    async run (client, message, args) {
+        if(message.channel.id == "500326922509615105")
+        {
+            // untested bcz idk JS
+            if(!talkedRecently.has(message.author.id)) 
+            {
+                message.channel.send('soup', {files:["./attachements/soop.mp4"]});
```

Comment:
Worried as you said this is untested because this doesn't look like the example for local files in the docs
https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=send

Reply:
Further googling suggests other people are using it like this and it works
🤷‍♂️